pac4j-core/src/main/java/org/pac4j/core/http/ajax/DefaultAjaxRequestResolver.java

```diff
@@ -14,11 +14,15 @@
 
     @Override
     public boolean isAjax(final WebContext context) {
-        final boolean xmlHttpRequest = AJAX_HEADER_VALUE.equalsIgnoreCase(context.getRequestHeader(AJAX_HEADER_NAME).orElse(null));
-        final boolean hasDynamicAjaxParameter = Boolean.TRUE.toString()
-            .equalsIgnoreCase(context.getRequestHeader(IS_AJAX_REQUEST).orElse(null));
-        final boolean hasDynamicAjaxHeader = Boolean.TRUE.toString()
-            .equalsIgnoreCase(context.getRequestParameter(IS_AJAX_REQUEST).orElse(null));
+        final boolean xmlHttpRequest = context.getRequestHeader(AJAX_HEADER_NAME)
```

Comment:
an alternative would be to chain flatmaps and map instead of using orElse but I agree this will look stranger than this... I suppose it's ok to keep it like this :)